.trunk/configs/.markdownlint.yaml

```diff
@@ -8,3 +8,5 @@ line_length: false
 spaces: false
 url: false
 whitespace: false
+no-duplicate-heading:
+  siblings_only: true
```

Comment:
Note this config for markdownlint is to help address the below

I want the ability to use the same header value nested in different levels
```
# v1.0
## Summary

# v2.0
## Summarty
```

We need to use 'siblings_only` for this ability.

Below is what I was seeing prior
$ trunk check ROADMAP.md               
Checking 100% [===========================================================================================================================>]  14/14  1.5s 
                                                                                                                                                          
  ISSUES  

ROADMAP.md:144:0
 144:0  low  Multiple headings with the same content  markdownlint/MD024
 148:0  low  Multiple headings with the same content  markdownlint/MD024
 161:0  low  Multiple headings with the same content  markdownlint/MD024
 165:0  low  Multiple headings with the same content  markdownlint/MD024
 172:0  low  Multiple headings with the same content  markdownlint/MD024
 176:0  low  Multiple headings with the same content  markdownlint/MD024